src/ensembl/src/shared/components/view-in-app/ViewInApp.test.tsx

```diff
@@ -1,18 +1,58 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
+import faker from 'faker';
 import { mount } from 'enzyme';
 
-import ViewInApp from './ViewInApp';
-import * as urlFor from 'src/shared/helpers/urlHelper';
+import { ViewInApp, AppButton, ViewInAppProps } from './ViewInApp';
+import ImageButton from 'src/shared/components/image-button/ImageButton';
 
-jest.mock('react-router-dom', () => ({
-  Link: () => <div>Link</div>
-}));
+import { ReactComponent as BrowserIcon } from 'static/img/launchbar/browser.svg';
+import { ReactComponent as EntityViewerIcon } from 'static/img/launchbar/entity-viewer.svg';
+
+const push = jest.fn();
+
+const renderComponent = (props: Partial<ViewInAppProps>) => {
+  const defaultProps = { links: {}, push };
+  const completeProps = {
+    ...defaultProps,
+    ...props
+  };
+
+  const renderedComponent = <ViewInApp {...completeProps} />;
+
+  return mount(renderedComponent);
+};
+
+const linkUrl = {
```

Comment:
I can't help feeling that the name should be in plural (linkUrls, and linkIcons below).